code/game/objects/items/weapons/twohanded.dm

```diff
@@ -248,6 +248,59 @@
 	sharp = IS_SHARP_ITEM_SIMPLE
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	attack_verb = list("attacked", "stabbed", "jabbed", "torn", "gored")
+	var/current_angle = 45	//Based on what direction the tip of the spear is pointed at in the sprite; maybe someone makes a spear that points northwest
+
+/obj/item/weapon/twohanded/spear/throw_at(atom/target, range, speed, thrower, spin, flying)
+	spin = FALSE
+	//Find the direction the spear is to be thrown, then rotate it based on that angle
+	var/rotation_value
+	switch(get_dir(thrower, target))
+		if(NORTH)
+			rotation_value = 0 - current_angle
+		if(NORTHEAST)
+			rotation_value = 45 - current_angle
+		if(EAST)
+			rotation_value = 90 - current_angle
+		if(SOUTHEAST)
+			rotation_value = 135 - current_angle
+		if(SOUTH)
+			rotation_value = 180 - current_angle
+		if(SOUTHWEST)
+			rotation_value = 225 - current_angle
+		if(WEST)
+			rotation_value = 270 - current_angle
+		if(NORTHWEST)
+			rotation_value = 315 - current_angle
```

Comment:
Why don't you just use get_angle?

Reply:
Did not know that existed. Is it more efficient? Comment says it's for beams, where I'd assume you want precision. Just aiming to get the general direction of the spear's throw.

Reply:
It gives you the exact angle, but is there a reason you wouldn't want it to be exactly aiming at your target?

Reply:
Simply a performance concern. Exact angle's not necessary for this process, which is why I went with get_dir. Using anything other than 45 degree angles would result in a funky sprite as well.